backend/src/nodes/nodes/external_stable_diffusion/outpainting.py

```diff
@@ -0,0 +1,271 @@
+            SliderInput(
+                "CFG Scale",
+                minimum=1,
+                default=7,
+                maximum=20,
+                controls_step=0.1,
+                precision=1,
+            ),
+            EnumInput(
+                ResizeMode,
+                default_value=ResizeMode.JUST_RESIZE,
+                option_labels=RESIZE_MODE_LABELS,
+            ),
+            SliderInput(
+                "Tile Width",
+                minimum=64,
+                default=512,
+                maximum=2048,
+                slider_step=8,
+                controls_step=8,
+            ),
+            SliderInput(
+                "Tile Height",
+                minimum=64,
+                default=512,
+                maximum=2048,
+                slider_step=8,
+                controls_step=8,
+            ),
+            SliderInput(
+                "Pixels to Expand",
+                minimum=8,
+                default=128,
+                maximum=256,
+                slider_step=8,
+                controls_step=8,
+            ).with_id(11),
+            SliderInput(
+                "Mask Blur",
+                minimum=0,
+                default=4,
+                maximum=64,
+            ),
+            BoolInput("Extend Left", default=True).with_id(13),
+            BoolInput("Extend Right", default=True).with_id(14),
+            BoolInput("Extend Up", default=True).with_id(15),
+            BoolInput("Extend Down", default=True).with_id(16),
+            EnumInput(
+                OutpaintingMethod, default_value=OutpaintingMethod.POOR_MAN_OUTPAINTING
+            ).with_id(17),
+            group(
+                "conditional-enum",
+                {
+                    "enum": 17,
+                    "conditions": [
+                        OutpaintingMethod.POOR_MAN_OUTPAINTING.value,
+                        OutpaintingMethod.OUTPAINTING_MK2.value,
+                        OutpaintingMethod.OUTPAINTING_MK2.value,
+                    ],
+                },
+            )(
+                EnumInput(InpaintingFill, default_value=InpaintingFill.FILL),
+                SliderInput(
+                    "Fall-off Exponent (lower=higher detail)",
+                    minimum=0,
+                    default=1,
+                    maximum=4,
+                    precision=2,
+                    slider_step=0.01,
+                    controls_step=0.01,
+                ),
+                SliderInput(
+                    "Color Variation",
+                    minimum=0,
+                    default=0.05,
+                    maximum=1,
+                    precision=2,
+                    slider_step=0.01,
+                    controls_step=0.01,
+                ),
+            ),
+        ]
+
+        # target_w = math.ceil((init_img.width + left + right) / 64) * 64
+        # target_h = math.ceil((init_img.height + up + down) / 64) * 64
```

Comment:
what are these for? are they still necessary?

Reply:
Ah, nope!  Note to myself.  I've removed it.